redata/checks/data_schema.py

```diff
@@ -65,6 +67,6 @@ def schema_to_dict(schema):
                 if curr_type != prev_type:
                     print (f"Type of column: {el} changed from {prev_type} to {curr_type}")
                     insert_schema_changed_record(table, 'column added', el, current_dict[el], len(current_dict))
```

Comment:
It's not part of your change really.. but shouldn't this message be updated?
```suggestion
                    insert_schema_changed_record(table, 'column changed', el, current_dict[el], len(current_dict))
```

Reply:
Yea... good point :) Updating that.